README.md

```diff
@@ -22,12 +22,12 @@ deposits must conform to [BagIt] format.
 
 To prepare the deposit for inclusion in EASY `easy-stage-dataset` performs the following tasks:
 
-1. It generates the metadata required for an EASY dataset:
-   * Administrative Metadata
-   * EASY Metadata (descriptive metadata)
-   * ...
-2. It stages a digital object to represent the entire dataset for ingest in Fedora, using the metadata generated in 1.
-3. It stages a digital object for each file and folder in the dataset for ingest in Fedora.
+ 1. It generates the metadata required for an EASY dataset:
+    * Administrative Metadata
+    * EASY Metadata (descriptive metadata)
+    * ...
+ 2. It stages a digital object to represent the entire dataset for ingest in Fedora, using the metadata generated in 1.
+ 3. It stages a digital object for each file and folder in the dataset for ingest in Fedora.
 
 The results of steps 1-3 can be ingested into the EASY Fedora Commons Repository.
 The command `easy-stage-file-item` executes step 3 to stage a file for ingestion into an existing dataset.
```

Comment:
* The only `easy-stage-file-item` use case for now is to add A/V files stored in dark archive in existing datasets.
* It will not work for folders. However, all missing folders (ancestors) will be staged.